scripts/DDL.sql

```diff
@@ -12,6 +12,11 @@ CREATE TABLE jobs.roles(
 	color text
 );
 
+CREATE TABLE jobs.qualifications(
+	id serial PRIMARY KEY,
+	name text UNIQUE
```

Comment:
Not sure about the exact flow, but why don't we give the qualifications an english identifier that will be more readable then a random number?  
 
Is there a specific reason (in application flow terms) for making it a sequence?

Reply:
yes, a sequence is a better practice for indexing